src/applications/my-education-benefits/tests/fixtures/data/form-submission-test-data.js

```diff
@@ -0,0 +1,142 @@
+export const submissionForm = {
```

Comment:
This is test form data for unit tests based on what I saw in memory with the debugger. This data will be parsed by transform and changed to meaningful JSON for the service.